lightning-invoice/src/utils.rs

```diff
@@ -633,6 +636,33 @@ fn filter_channels<L: Deref>(
 		.collect::<Vec<RouteHint>>()
 }
 
+pub(crate) fn address_from_fallback(fallback: &Fallback, currency: Currency) -> Address {
```

Comment:
Once we have the enum changed, we probably don't need a util. Could you test the behavior using the `Invoice` interface in `lib.rs`?

Reply:
just testing one address now in the big invoice megatest, let me know if you want another test that tests multiple